packages/ui-components/src/components/text-field/text-field.tsx

```diff
@@ -0,0 +1,69 @@
+import { Component, Event, EventEmitter, h, Host, Prop } from '@stencil/core';
+import { EInputFieldType, IErrorMsgs } from './text-field.types';
+
+@Component({
+	tag: 'kv-text-field',
+	styleUrl: 'text-field.scss',
+	shadow: true
+})
+export class KvTextField {
+	/** (optional) Text field tyoe */
+	@Prop({ reflect: true }) type!: EInputFieldType;
+	/** (optional) Text field label */
+	@Prop({ reflect: true }) label: string;
+	/** (optional) Text field input name */
+	@Prop({ reflect: true }) inputName: string;
+	/** (optional) Text field place holder */
+	@Prop({ reflect: true }) placeholder: string;
+	/** (optional) Text field disabled */
+	@Prop({ reflect: true }) disabled = false;
+	/** (optional) Text field required */
+	@Prop({ reflect: true }) required = false;
+	/** (optional) Text field loading state */
+	@Prop({ reflect: true }) loading = false;
+	/** (optional) Text field help text */
+	@Prop({ reflect: true }) helpText: string;
+	/** (optional) Text field error messages */
+	@Prop({ reflect: true }) errorMessages: IErrorMsgs;
```

Comment:
these are not being used anywhere, they seem to be related to form validation, either remove them or discuss an option with the team to project content into this component.